module/metrics/network.go

```diff
@@ -82,7 +82,7 @@ func NewNetworkCollector(opts ...NetworkCollectorOpt) *NetworkCollector {
 			Namespace: namespaceNetwork,
 			Subsystem: subsystemGossip,
 			Name:      nc.prefix + "duplicate_messages_dropped",
-			Help:      "number of duplicate messages dropped",
+			Help:      "number of duplicate messages inboundDropped",
```

Comment:
```suggestion
			Help:      "number of duplicate messages dropped",
```
There are several other instances of this bad auto-rename in this file.

Reply:
oops, fixed those, thanks for spotting